### PR TITLE
Ui homepage [Finishes #158299469]

### PR DESCRIPTION
**Pull request by wandesky**
Not so good now.
avatar

### CODE REVIEW

**Comment by wandesky-tests** [comment]
Everything looks good. Nice avatar by the way.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # uwanjani
 Playing around with git 
 
-##Task 1
+## Task 1
 Songesha branch ui mbele ya master (kidogo tu)
+
+## Task 2
+Make a new branch from ui. Call it ui-homepage
+I made this line from ui-homepage
```
Comment (wandesky-tests) [inline README.md:9]: Maybe you should add that you intend to merge ui-homepage to ui

**Review by wandesky-tests — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (wandesky) [reply README.md:9]: Thank you @wandesky-tests but I think I will keep it that way. One step at a time. 😞

**Comment by wandesky** [comment]
Now that I have received reviews, I am going to merge the two branches.

**Pull request merged**